prov/gni/src/gnix_nic.c

```diff
@@ -45,7 +45,7 @@
 #include "gnix_mbox_allocator.h"
 
 static int gnix_nics_per_ptag[GNI_PTAG_USER_END];
-static DLIST_HEAD(gnix_nic_list);
+DLIST_HEAD(gnix_nic_list);
```

Comment:
You cannot extern statics, and I need access to the global nic_list allow us to progress all the nics.